src/app/Components/Button.svelte

```diff
@@ -0,0 +1,95 @@
+<script lang="ts">
+  // props
+  let color: string = "grey"; // optional
+  let isOutlined: boolean = false; // optional
+  let hasDropShadow: boolean = false; // optional
+  let size: string = "medium"; // optional
+  export let text: string = ""; // required
+  export let callbackFunction: any = null; // required
+
+  export let className: string = "";
+
+  /**
+   * Creates the class name for the button if the styling props are passed
+   *
+   * @return {void}
+   */
+  const createStyles = (): void => {
+    className += color;
+    className += " " + size;
+    isOutlined && (className += " outlined");
+    hasDropShadow && (className += " shadow");
+  };
+
+  createStyles();
+</script>
+
+<style>
+  button {
+    margin: 0px 10px;
+    border-radius: 10px;
+    text-transform: uppercase;
+    font-family: Cabin, sans-serif;
+    font-size: 15px;
+    font-weight: bold;
+    transition-duration: 0.2s;
+  }
+
+  button:hover {
+    cursor: pointer;
+  }
+
+  .small {
+    padding: 5px 10px;
+  }
+
+  .medium {
+    padding: 10px 20px;
+  }
+
+  .large {
+    padding: 20px 40px;
```

Comment:
**[Question]**: I am not sure where we have landed, if we have decided to implement utility classes, we shouldn't have these in local styling.